src/pages/admin/users/create-user-dialog-form.jsx

```
@@ -0,0 +1,54 @@
+import { Fragment } from 'react';
+import * as Yup from 'yup';
+import { CreateDialogForm } from '@/components/dialog-form';
+import { InputField, SelectField } from '@/components/field';
+import { GENDER, REGEX_PATTERN } from '@/constant';
+import { capitalizeStr } from '@/utils/string';
```

ESLint

[error] 'capitalizeStr' is defined but never used.

(no-unused-vars)